### PR TITLE
Add new blogpost about quick booting

### PR DESCRIPTION
**Pull request by Vogtinator**
Doesn't quite fit the theme or the style of the other blog posts, but is hopefully still a good addition.

### CODE REVIEW

File: blog/_posts/2020-02-04-quickboot.md [hunk 1]
```diff
@@ -0,0 +1,542 @@
...
+```
+
+`btrfsmaintenance-refresh.service` is a bit special: It calls `systemctl` during execution
+to enable/disable and start/stop the btrfs-\*.timer units. Those depend on `time-sync.target`, which itself needs `network.service` through `chronyd.service`.
+`wicked.service` is the next item on the list. Before the unit is considered `active`, it tries
+to fully configure and setup all configured interfaces, which includes DHCPv4 and v6 by default.
+This is directly used as state for `network.service` and thus `network.target`.
+There is no distinction between `network.service` and `network-online.target` by wicked.
+To make the bootup quicker, switching to NetworkManager is an option,
+which interprets `network.service` in a more async way and thus is much quicker to into the `active` state.
```
Comment (laenion) [inline blog/_posts/2020-02-04-quickboot.md]: A word seems to be missing here, probably "change".

**Comment by Vogtinator** [comment]
Forgot to add: My plan is to move `home:favogt:quickboot` into `devel:kubic:quickboot` or somewhere else before publishing the article as using `home:` repos isn't that great. Opinions?

**Comment by sysrich** [comment]
> Forgot to add: My plan is to move `home:favogt:quickboot` into `devel:kubic:quickboot` or somewhere else before publishing the article as using `home:` repos isn't that great. Opinions?

Yeah lets move it to devel:kubic:quickboot then publish it with that more official path :)

**Comment by Vogtinator** [comment]
I copied the project into `devel:kubic:quickboot` now and updated the references. Not built yet, so repo is not available yet.

**Pull request merged**